src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -89,7 +90,7 @@ public static function parse($input, $flags = 0)
      *
      * @return string A YAML string representing the original PHP array
      */
-    public static function dump($array, $inline = 2, $indent = 4, $flags = 0)
+    public static function dump($array, $inline = 2, $indent = 4, $absIndent = 0,$flags = 0)
```

Comment:
Do we actually need to expose it to the outside here ? This makes the signature harder to use and understand (and a pain for backward compatibility).

Btw, the fact that you are adding the argument before an existing one breaks all usages of the flags.